### PR TITLE
[CNVS Upgrade: Stylelint 2 of 3] Fix stylelint warnings

### PR DESCRIPTION
**Pull request by jfurrow**
This PR adjusts the formatting of all of our styles to match the stylelint rules defined in ~~#1092~~.

### CODE REVIEW

**Comment by philipnrmn** [comment]
I can't get this to build locally, I'm seeing errors related to the `reactjs-components` import in `index.less`

**Comment by jfurrow** [comment]
@philipnrmn You'll need to have the `cnvs-upgrade/master` branch checked out in `reactjs-components`, and then locally link the npm module. This should help: https://github.com/dcos/dcos-ui/blob/master/CONTRIBUTING.md

File: package.json [hunk 1]
```diff
@@ -102,6 +102,8 @@
     "source-map-loader": "0.1.5",
     "string-replace-webpack-plugin": "0.0.3",
     "style-loader": "0.13.1",
+    "stylelint-config-standard": "^13.0.0",
```
Comment (MatApple) [inline package.json]: Caret

Comment (jfurrow) [inline package.json]: Fixed!

**Comment by MatApple** [comment]
After shrink-wrapping this LGTM!

**Comment by jfurrow** [comment]
@MatApple I updated the shrinkwrap, it should be good now!

**Pull request merged**